hypershift-operator/controllers/nodepool/inplace.go

```diff
@@ -21,13 +27,40 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/cache"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+const (
+	// CurrentMachineConfigAnnotationKey is used to fetch current targetConfigVersionHash
+	CurrentMachineConfigAnnotationKey = "machineconfiguration.openshift.io/currentConfig"
+	// DesiredMachineConfigAnnotationKey is used to indicate the version a node should be updating to
+	DesiredConfigAnnotationKey = "machineconfiguration.openshift.io/desiredConfig"
+	// MachineConfigDaemonStateAnnotationKey is used to fetch the state of the daemon on the machine.
+	MachineConfigDaemonStateAnnotationKey = "machineconfiguration.openshift.io/state"
+	// MachineConfigDaemonStateWorking is set by daemon when it is applying an update.
+	MachineConfigDaemonStateWorking = "Working"
+	// MachineConfigDaemonStateDone is set by daemon when it is done applying an update.
+	MachineConfigDaemonStateDone = "Done"
+	// MachineConfigDaemonStateDegraded is set by daemon when an error not caused by a bad MachineConfig
+	// is thrown during an update.
+	MachineConfigDaemonStateDegraded = "Degraded"
+	// MachineConfigDaemonReasonAnnotationKey is set by the daemon when it needs to report a human readable reason for its state. E.g. when state flips to degraded/unreconcilable.
+	MachineConfigDaemonReasonAnnotationKey = "machineconfiguration.openshift.io/reason"
+	// DesiredDrainerAnnotationKey is set by the MCD to indicate drain/uncordon requests
+	DesiredDrainerAnnotationKey = "machineconfiguration.openshift.io/desiredDrain"
+	// LastAppliedDrainerAnnotationKey is set by the controller to indicate the last request applied
+	LastAppliedDrainerAnnotationKey = "machineconfiguration.openshift.io/lastAppliedDrain"
+	// DrainerStateDrain is used for drainer annotation as a value to indicate needing a drain
+	DrainerStateDrain = "drain"
```

Comment:
DrainerStateDrain seems unused

Reply:
Will eventually be needed once we implement draining, but can remove for now